### ISSUE TITLE
#[derive(WrapperApi)] fails if `use std::io::Result;`

### ISSUE THREAD

**Issue by mozkeeler** [szymonwieloch/rust-dlopen#5]
This fails to compile for me:
```
#[macro_use]                                                                                         
extern crate dlopen_derive;                                                                          
extern crate dlopen;                                                                                 
use dlopen::wrapper::WrapperApi;                                                                     
                                                                                                     
use std::io::Result;                                                                                 
                                                                                                     
#[derive(WrapperApi)]                                                                                
struct Example {                                                                                     
    do_something: extern "C" fn(),                                                                   
}                                                                                                    
                                                                                                     
fn main () {                                                                                         
}
```
The error is:
```
error[E0244]: wrong number of type arguments: expected 1, found 2
 --> src/main.rs:8:10
  |
8 | #[derive(WrapperApi)]
  |          ^^^^^^^^^^ expected 1 type argument

error: aborting due to previous error
```
(if I take out the `use std::io::Result;` line, it compiles)

**Comment by szymonwieloch** [comment]
@mozkeeler Hi! Thank you for the report. I think that I know what happens: there is a name conflict between two Result-s: the "normal" one and the one that you included to the file. I guess that I need to fix the code generation and use global references to types. A workaround is to not include any types that are called "Result" or simply rename it like this:

```
use std::io::Result as IoResult;
```

 I am quite busy at the moment but withing two weeks I should create a fix for this bug.

**Comment by mozkeeler** [comment]
Thanks! No worries about fixing it super fast - I mostly filed this in case anyone else had run into the same problem (I found the error message confusing, although really I suppose that's a compiler issue...)

**Comment by szymonwieloch** [comment]
@mozkeeler Fixed in the version 0.1.4.

**Comment by mozkeeler** [comment]
Thanks!
